docs/core/deploying/native-aot/index.md

```diff
@@ -25,37 +25,82 @@ The Native AOT deployment model uses an ahead-of-time compiler to compile IL to
 - The compiler toolchain and developer packages for libraries that the .NET runtime depends on.
 - Alpine (3.15+)
 
+  Publish dependencies:
+
+  ```sh
+  sudo apk add clang build-base icu-dev
```

Comment:
Does publishing really need `icu-dev`? We should be loading ICU dynamically.

Reply:
You are right. We only need `icu-libs` on Alpine. I think the only reason we would need icu-dev is if we are using `StaticICULinking` https://github.com/dotnet/runtime/blob/088741956a4a9d15c4dcbbc81f8e7e25fc59c161/src/coreclr/nativeaot/docs/compiling.md#using-statically-linked-icu (which is an advanced scenario).

```suggestion
  sudo apk add clang build-base icu-libs
```